website/start/routes.ts

```diff
@@ -42,6 +42,46 @@ router
       .as('actions:auth.register')
       .use(middleware.guest())
 
+    router
+      .on('/password/forgot')
+      .renderInertia('auth/forgot-password/index')
+      .as('pages:auth.forgot-password')
+      .use(middleware.guest())
+
+    router
+      .on('/password/forgot/sent')
+      .renderInertia('auth/forgot-password/sent')
+      .as('page:auth.forgot-password.sent')
+      .use(middleware.guest())
+
+    router
+      .post('/password/forgot/new', [AuthenticationController, 'sendForgotPassword'])
+      .as('actions:auth.forgot-password.send')
+      .use(middleware.guest())
+
+    router
+      .route(
+        '/password/forgot/callback',
+        ['GET', 'POST'],
+        [AuthenticationController, 'callbackForForgotPassword']
+      )
+      .as('actions:auth.forgot-password.callback')
+      .middleware([middleware.verifyUrlSignature(), middleware.automaticSubmit()])
```

Comment:
Here you probably don't want an automatic submit because on GET you show the reset form and on POST you reset the password.
That way, both POST and GET are authenticated because the URL is the same and the signature is the same.